# Michael M. Hoffman

101 College St TMDT 11-311\
Toronto, ON M5G 1L7

Work +1 416 581 7481\
E-mail michael.hoffman@utoronto.ca

## Current positions

2013–present Princess Margaret Cancer Centre *Toronto, ON*

Scientist

2014–present University of Toronto *Toronto, ON*

Assistant Professor**, Department of Medical Biophysics**

Assistant Professor**, Department of Computer Science**

## Education

2008–2013 University of Washington *Seattle, WA*

Senior Fellow

* Mentor: Dr. William Stafford Noble, Professor

* National Human Genome Research Institute K99/R00 Pathway to
  Independence Award

2003–2008 University of Cambridge (Trinity College) *Cambridge, UK*

Doctor of Philosophy, Bioinformatics

* Adviser: Dr. Ewan Birney, Associate Director, European Bioinformatics
  Institute

* Marshall Scholar

* National Science Foundation Graduate Research Fellow

1998–2003 The University of Texas at Austin *Austin, TX*

Bachelor of Science, Biochemistry, with Highest Honors, Special Honors

Bachelor of Arts, Plan II Honors Liberal Arts, with Highest Honors

* Adviser: Dr. Andrew D. Ellington, Professor

* College of Natural Sciences Commencement Speaker, Dean’s Honored
  Graduate

* College of Liberal Arts Dean’s Distinguished Graduate

* GPA: 3\.965/4\.000—University Honors Major GPA: 4\.000/4\.000

## Publications (journal)

* > News and Views: Muerdter F, Stark A. *Nature*. 2014 Aug 28;
  > 512:374–5\.

* > Featured Article in *Nucleic Acids Research*.

* > Cited 103 times.

* > Author contribution: “Lead analyst”.

* > Featured on cover of *Nature*.

* > Recommended by Faculty of 1000\.

* > News and Views: Ecker JR et al. *Nature.* 2012 Sep 6; 489:52–55\.

* > Cited 2,809 times.

* > Recommended by Faculty of 1000\.

* > Cited 296 times.

* > Cited 129 times.

* > Recommended by Faculty of 1000\.

* > Cited 648 times.

* > Featured on cover of *Nature*.

* > News and Views: Schmutz J, Grimwood J. *Nature*. 2004 Dec 9;
  > 432:679–80\.

* > Cited 1,552 times.

* > Cited 96 times.

## Publications (conference proceedings)

## Publications (submitted)

## Other scholarships and awards

* Canadian Cancer Society Research Institute Junior Investigator Grant
  Panel Travel Award, 2014\.

* Trinity College Moore, Beale Sargent and Mitchell Fund, 2005

* The University of Texas at Austin Junior Fellows Research Grant,
  2003\.

* The University of Texas at Austin Undergraduate Research Fellowship,
  1999-2000, 2000-2001, 2002-2003\.

* NSF Research Experience for Undergraduates supplement, 1999\.

* Dedman Merit Scholarship, 1998-2002\.

* Dorothy B. Banks Charitable Trust Scholarship, 1999-2000\.

* IBM Thomas J. Watson Memorial Scholarship, 1998-2002\.

* National Merit Scholarship, 1998-2002\.

## Honors

* Bioinformatics.ca Bioinformatics Expert, 2014\.

* *Genome Technology* Young Investigator, 2011\.

* Phi Beta Kappa Award of Distinction, 2003\.

* Phi Beta Kappa Induction, 2003\.

* Cambridge Overseas Trust Honorary Scholar, 2003\.

* College of Liberal Arts Junior Fellow, 2000–2003\.

* College of Natural Sciences Dean’s Scholar, 1998–2003\.

* College of Liberal Arts Plan II Honors Program (2% of university),
  1998–2003\.

* College of Natural Sciences College Scholar 2000, 2001, 2002, 2003\.

## Invited/keynote presentations

* Semi-automated human genome annotation using chromatin data. Institute
  for Operations Research and the Management Sciences Annual Meeting.
  Philadelphia, PA. 1–4 November 2015\.

* Cold Spring Harbor Laboratory Meeting on Genome Informatics. Cold
  Spring Harbor, NY. October 2015\.

* Transcription factor binding motifs in an expanded epigenetic
  alphabet. Taiwan-Canada Joint Workshop on Epigenetics. Canadian Human
  and Statistical Genetics Meeting. Vancouver, BC. 21 April 2015\.

* Semi-automated genome annotation and an expanded epigenetic alphabet.
  Program in Genetic Networks Workshop. Canadian Institute for Advanced
  Research. Toronto, ON. 10 April 2015\.

* Semi-automated genome annotation and an expanded epigenetic alphabet.
  Department of Biochemistry and Molecular Genetics. University of
  Colorado Anschutz Medical Campus. Denver, CO. 3 April 2015\.

* Semi-automated genome annotation and an expanded epigenetic alphabet.
  Bioinformatics Club. University of Waterloo. Waterloo, ON. 24 March
  2015\.

* Semi-automated human genome annotation. Signals and Intelligence
  Chapter/Signal Processing Chapter/Engineering in Medicine and Biology
  Chapter. Toronto Section. Institute of Electrical and Electronics
  Engineers. Toronto, ON. 9 March 2015\.

* Transcription factor binding prediction. Biohackathon. University of
  Toronto Internationally Genetically Engineered Machine. University of
  Toronto. Toronto, ON. 22 January 2015\.

* Semi-automated genome annotation and an expanded epigenetic alphabet.
  Understanding non-coding DNA through intra and inter-species
  epigenomic variation. Bellairs Research Institute. Holetown, Barbados.
  9 January 2015\.

* Semi-automated genome annotation and an expanded epigenetic alphabet.
  McGill University. Montreal, QC. 21 November 2014\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Major Discoveries in Biology. Undergraduate Program in
  Genomic Sciences. National Autonomous University of Mexico. Mexico
  City, Mexico (videoconference). 5 March 2014\.

* Semi-automated Genome Annotation with Segway. Institute of Electrical
  and Electronics Engineers International Workshop on Genomic Signal
  Processing and Statistics 2013\. Houston, TX. 17 November 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Toronto Bioinformatics Users Group. Toronto, ON. 27
  November 2013\.

* Unsupervised pattern discovery in human chromatin structure through
  genomic segmentation. Workshop on Epigenomics and Cell Function 2013\.
  Association for Computing Machinery Conference on Bioinformatics,
  Computational Biology and Biomedicine. Washington, DC. 22 September
  2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Genentech. South San Francisco, CA. 12 November 2012\.

* Unsupervised pattern discovery in human chromatin data. Computational
  Biology Center Guest Seminar. Memorial Sloan-Kettering Cancer Center.
  New York, New York. 18 May 2011\.

* Segway: finding patterns in chromatin data. Center for Bioinformatics
  Research Special Talk, Indiana University. Bloomington, Indiana. 18
  November 2010\.

* Finding the patterns in chromatin data. Fifth Barbados Workshop on
  Gene Regulation: The role of chromatin in 3D structure. Holetown,
  Barbados. 21 April 2010\.

* Properties of natural selection in mammalian promoters. Weizmann UK
  Symposium: Biological complexity: from models to systems. London,
  England. Summer 2008\.

* Predicting Selection in Promoters by Simulating the Effects of
  Mutations. Waterman Seminar, Leibniz Institute of Plant Genetics and
  Crop Plant Research. Gatersleben, Germany. Autumn 2007\.

* Student address, The University of Texas at Austin 120th Spring
  Commencement, College of Natural Sciences Convocation. Austin, TX. 17
  May 2003\.

* Student address, The University of Texas at Austin 55th Honors Day
  Convocation. Austin, TX. 12 April 2003\.

## Session chair

* Big data analytics in genomics. Institute for Operations Research and
  the Management Sciences Annual Meeting. Philadelphia, PA. November
  2015\.

* Epigenomics and non-coding genome. Cold Spring Harbor Laboratory
  Meeting on Genome Informatics. Cold Spring Harbor, NY. October 2015\.

* Genomes and big data – data integration. Fourth Annual Canadian Human
  and Statistical Genetics Meeting. Vancouver, BC. 19 April 2015\.

* Genomic and epigenomic signatures in cancer. Epigenetic Mechanisms in
  Cancer. Structural Genomics Consortium. Toronto, ON. 4 June 2014\.

* Genome organization and variant detection. Association for Computing
  Machinery Conference on Bioinformatics, Computational Biology and
  Biomedicine. Washington, DC. 24 September 2013\.

## Invited panelist

* Informatics, Annotation, and Computing. Applied Cancer Genomics
  Symposium, Toronto, ON. 3 October 2014\.

* The job search process: postdoctoral, faculty, and industry jobs.
  Toronto Bioinformatics Users Group. Toronto, ON. 27 November 2013\.

* Networking. Women in Genome Sciences Discussion. University of
  Washington. 23 February 2012\.

## Selected conference presentations

* Segway: semi-automated genome annotation. Bioinformatics Open Source
  Conference. Dublin, Ireland. 10 July 2015\.

* Transcription factor binding in an expanded epigenetic alphabet. World
  Congress on Medical Physics. International Union for Physical and
  Engineering Sciences. Toronto, ON. 11 June 2015\.

* Transcription factor binding in an expanded epigenetic alphabet.
  Keystone Symposium on DNA Methylation. Keystone Symposia on Molecular
  and Cellular Biology. Keystone, CO. 1 April 2015\.

* Transcription factor motif discovery and prediction in an expanded
  epigenetic alphabet. Cold Spring Harbor Laboratory/Wellcome Trust
  Conference on Genome Informatics. Cambridge, United Kingdom. 22
  September 2014\.

* One genome, many epigenomes, machine learning. Workshop on Big Data
  for Health, University of Toronto. Toronto, ON. 4 July 2014\.

* Semi-automated annotation of functional elements with whole genome
  RNA-seq. Machine Learning in Computational Biology 2012\. Neural
  Information Processing Systems Conference 2012\. Stateline, NV. 7
  December 2012\.

* Semi-automated annotation of the human genome using chromatin data.
  University of Washington Postdoctoral Association Research Symposium.
  Seattle, WA. 27 November 2012\.

* Functional genomics and machine learning. SynBioCon 2012\. Austin, TX.
  26 May 2012\.

* Semi-automated annotation of the human genome using whole-genome
  RNA-seq. ENCODE and modENCODE AWG/PI Meeting. Cambridge, MA. 22 May
  2012\.

* Discovery of functional elements using whole-genome RNA-seq. The
  Biology of Genomes. Cold Spring Harbor, NY. 9 May 2012\.

* Chromosomal features: large-scale integration. ENCODE AWG Workshop.
  Stanford, CA. 7 March 2011\.

* Large-scale integration group update. ENCODE AWG Workshop. Barcelona,
  Spain. 20 July 2010\.

* Large-scale behavior task group. ENCODE AWG Workshop. Rockville, MD. 8
  March 2010\.

* Segway: a dynamic Bayesian network for genomic segmentation. ENCODE
  and modENCODE Consortia Meeting. Bethesda, MD. Spring 2009\.

* ENCODE and Segway. ENCODE AWG Workshop. Rockville, MD. December 2008\.

* Properties of natural selection in mammalian promoters.
  Sanger-Cambridge PhD Symposium. Hinxton, England. Spring 2008\.

* Sunflower: a model of transcription factor binding. EMBL Predoctoral
  Fellow Retreat. Barcelona, Spain. Summer 2007\.

* Measurements of evolution and natural selection. Trinity College
  Science Society Symposium. Cambridge, England. Spring 2007\.

* Poly: rapid development of embarrassingly parallelizable applications.
  Bioinformatics Open Source Conference. Glasgow, Scotland. Summer
  2004\.

* Visualizing protein–DNA and protein–RNA interactions with AANT.
  Trinity College Science Society Symposium. Cambridge, England. 13
  March 2004\.

## Other teaching

* “Epigenomics.” Summer Student Program. Department of Medical
  Biophysics. University of Toronto. 8 June 2015\.

* “Gene Regulation Network Analysis.” Pathway and Network Analysis of
  –omic Data. 3 June 2015\.

* MBP 1015Y: Biophysics Seminar. Department of Medical Biophysics.
  University of Toronto. 15 October 2014–13 May 2015\.

* “Gene Regulation Network Analysis.” High-throughput Biology: From
  Sequence to Networks. Canadian Bioinformatics Workshop. 3 May 2015\.

* "Transcription Factor Regulatory Analysis." Pathway and Network
  Analysis of omics Data. Canadian Bioinformatics Workshop, 4 June
  2014\.

* “Introduction to ENCODE” and “The Segway annotation of the ENCODE
  Data.” Iowa Institute of Human Genetics Bioinformatics Short Course.
  Taught two one-hour lectures. University of Iowa. 30–31 July 2013\.

* “Genes and gambling: using probability to make sense of biology.”
  Guest lecture for undergraduate computer science course, Bellevue
  College. Winter 2010\.

* “Introduction to Python.” Taught two-day seminar. European Molecular
  Biology Laboratory. Autumn 2005\.

* EMBL Predoctoral Bioinformatics Workshop. Organizing committee member
  and teaching assistant for three-day course. European Molecular
  Biology Laboratory. Autumn 2004\.

* “Biological sequence analysis.” Discussion leader for eight-week
  seminar. The University of Texas at Austin Society for Computational
  Biology. Summer 2003\.

## Other seminars

* Epigenetics. Princess Margaret Cancer Foundation Board Retreat.
  Toronto, ON. 8 November 2014\.

* Machine Learning and Epigenomics. Princess Margaret Cancer Centre
  Research Retreat. Huntsville, ON. 15 May 2014\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Department of Genetics. University of Pennsylvania.
  Philadelphia, PA. 25 July 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Ontario Cancer Institute. Toronto, ON. 4 July 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Institute for Human Genetics. University of California,
  San Francisco. San Francisco, CA. 21 March 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Analytical and Translational Genetics Unit.
  Massachusetts General Hospital. 24 April 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Department of Electrical and Computer Engineering.
  University of Texas at Austin. 22 April 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. The Jackson Laboratory. Bar Harbor, ME. 11 March 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. McKusick-Nathans Institute of Genetic Medicine. Johns
  Hopkins University. Baltimore, MD. 7 March 2013\.

* Semi-automated annotation of the human genome using chromatin data.
  Biomedical Postdoc Seminar. University of Washington. Seattle, WA. 4
  March 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Department of Human Genetics. University of Utah. Salt
  Lake City, UT. 25 February 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. National Eye Institute. National Institutes of Health.
  Bethesda, MD. 30 January 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Department of Molecular and Medical Genetics. Oregon
  Health and Science University. Portland, OR. 23 January 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Department of Human Genetics. Emory University. Atlanta,
  GA. 14 January 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. McDermott Center for Human Growth and Development/Center
  for Human Genetics. University of Texas Southwestern Medical Center.
  Dallas, TX. 7 January 2013\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Earl Stadtman Symposium on Genetics, Genomics, and
  Systems Biology. Division of Intramural Research. National Institutes
  of Health. Bethesda, MD. 17 December 2012\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Computational Biology Group Meeting/CSE 591C. Department
  of Computer Science. Seattle, WA. 14 December 2012\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Program in Bioinformatics and Integrative Biology.
  University of Massachusetts Medical School. Worcester, MA. 10 December
  2012\.

* Semi-automated annotation of the human genome using chromatin and
  RNA-seq data. Department of Pediatrics. University of California, San
  Diego. 4 June 2012\.

* Unsupervised and semi-supervised pattern discovery in human chromatin\
  structure. Section of Genetic Medicine. University of Chicago.
  Chicago, IL. 23 April 2012\.

* Simultaneous segmentation of multiple functional genomics data\
  sets with heterogeneous patterns of missing data. Department of
  Computer Science. Colorado State University. Fort Collins, CO. 10
  April 2012\.

* Semi-automated annotation of the human genome using whole-genome
  RNA-seq. Postdoctoral Research Talk. Department of Genome Sciences.
  University of Washington. 16 May 2012\.

* Segway: pattern discovery in multitrack functional genomics data.
  Research Reports. Department of Genome Sciences. University of
  Washington. 13 November 2009\.

* Segway: a dynamic Bayesian network for genomic segmentation. Reading
  and Research in Computational Biology. CSE 590C. 4 May 2009\.

* Properties of natural selection in mammalian promoters. Seminar, Broad
  Institute of MIT and Harvard. Cambridge, MA. Spring 2008\.

* Properties of natural selection in mammalian promoters. Seminar,
  Program in Bioinformatics and Integrative Biology, University of
  Massachusetts Medical School. Worcester, MA. Spring 2008\.

## Prior positions

* **Senior Fellow**, Department of Genome Sciences, University of
  Washington, 2008–2013\.

* **Predoctoral Fellow**, EMBL–European Bioinformatics Institute,
  2003–2008\.

* **Research Assistant**, Department of Chemistry and Biochemistry, The
  University of Texas at Austin, 1999–2003\.

* **Undergraduate Research Program Participant**, Cold Spring Harbor
  Laboratory, 2001\.

* **Undergraduate Research Fellow**, Plant Biochemistry Research
  Training Center, Washington State University, 2000\.

## Trainees and staff supervised

## Software

* Segway: a dynamic Bayesian network method for segmenting genomic data.
  \<http://pmgenomics.ca/hoffmanlab/proj/segway/\>.

* Segtools: exploratory data analysis of genomic segmentations.
  \<http://pmgenomics.ca/hoffmanlab/proj/segtools/\>.

* Genomedata: a format for storing large-scale functional genomics data.
  \<http://pmgenomics.ca/hoffmanlab/proj/genomedata/\>.

* Sunflower: a model of transcription factor binding and evolution.

## Databases

* AANT: the Amino Acid-Nucleotide Interaction Database.

* FIAT: Factors in *Arabidopsis* Translation.
  \<http://browning.cm.utexas.edu/arabidopsis/fiat/\>.

* AtProbe: *Arabidopsis thaliana* Promoter Binding Element Database.
  \<http://exon.cshl.org/cgi-bin/atprobe/atprobe.pl\>.

## Leadership and committee positions

* **Member**, Graduate Admissions Committee, Department of Computer
  Science, University of Toronto, 2014–2015\.

* **Member**, Program Committee, Research in Computational Molecular
  Biology (RECOMB), 2014–present.

* **Teleconference member,** Genomics Peer Review Committee, Canadian
  Institutes for Health Research, 2015\.

* **Member**, Review Committee, Collaborative Personalized Cancer
  Medicine Team Grant, Princess Margaret Cancer Centre, 2015\.

* **Judge**, Summer Student Poster Day, Department of Medical
  Biophysics, University of Toronto, 2014–2015\.

* **Mentor**, National Resource for Network Biology, Google Summer of
  Code, 2014\.

* **Co-chair**, University Health Network Cancer Genomics Program
  Bioinformatics Subcommittee, 2013–present.

* **Member**, Princess Margaret Genomics Centre Advisory Board,
  2013–present.

* **Chair**, Large-scale Integration Task Group, ENCODE Analysis Working
  Group, 2010–2012\.

* **Chair**, Cambridge University Student Pugwash Society, 2006–2007\.
  **Secretary**, 2007–2008\. *Organized seminar series on science and
  society topics.*

* **Representative**, EMBL–European Bioinformatics Institute Predoctoral
  Fellow Association, 2004–2005\.

* **Secretary/Treasurer**, International Society for Computational
  Biology Student Council, 2004\. **Member**, 2003–2004\.

* **President**, Texas Student Publications Board of Operating Trustees,
  2002–2003\. **Vice** **President**, 2001–2002\. **Member**,
  2000–2003\. *Managed finances, personnel, and policy of daily
  newspaper, radio station, television station, and other media. $2\.9
  million annual budget.*

* **Member**, College of Natural Sciences Dean’s Scholars Committee,
  2000–2003\.

* **Founder** and **Co-President**, University of Texas at Austin
  Barbecue Club, 1999–2003\.

* **Chair**, Texas Student Publications Handbook Revision Committee,
  2002\.

* **Chair**, Texas Student Publications Election Committee, 2001–2002\.

* **Member**, University of Texas at Austin Information Technology
  Coordinating Council, 1999–2002\.

* **Member**, University of Texas at Austin General Faculty Admissions
  and Registration Committee, 1999–2000\.

## Professional activities

* **Organizer**, Early Career Investigators Meeting on Quantitative
  Problems in Human Genetics and Health. Banff International Research
  Station. Banff, AB, Canada. 10 January 2016–15 January 2016\.

* **Reviewer** for journals: Nature Methods, Genome Biology, PLOS
  Computational Biology, Nucleic Acids Research, Scientific Reports,
  Bioinformatics, PLoS ONE, BMC Genomics, BMC Bioinformatics, Journal of
  Machine Learning Research, IEEE Transactions on Computational Biology
  and Bioinformatics, Artificial Intelligence in Medicine, Computational
  and Structural Biotechnology Journal.

* **Reviewer** for conferences: Intelligent Systems in Molecular
  Biology/European Conferences on Computational Biology (ISMB/ECCB),
  Research in Computational Molecular Biology (RECOMB).

* **Reviewer** for funders: United Kingdom Medical Research Council.

* **Reviewer** for research theses: Department of Computer Science,
  University of Toronto (Aryan Arbabi).

* **Invited Participant**, National Human Genome Research Institute
  planning workshop: Planning the future of genomics: foundational
  research and applications in genomic medicine.

## Other organizations

* **Member**, Global Alliance for Genomics and Health, 2014–present.

* **Member**, Association for Computing Machinery Special Interest Group
  on Bioinformatics, 2013–present.

* **Member**, National Postdoctoral Association, 2011–2013\.

* **Member**, Northwest Institute of Genetic Medicine, 2009–2013\.

* **Member**, University of Washington Postdoctoral Association,
  2008–2013\.

* **Member**, Phi Beta Kappa, 2003–present.

* **Member**, International Society for Computational Biology,
  2003–2004, 2010–2011, 2015–2016\.

* **Junior Member**, Isaac Newton Institute for Mathematical Sciences,
  2006–2013\.

* **Member**, American Association for the Advancement of Science,
  2007–2008\.

* **Member**, Genetics Society, 2006–2007\.

* **Member**, Society of Computational Biology, 2002–2003\.

* **Member**, Microbiology and Molecular Biology Student Society,
  1998–1999\.

## Interviews 

* “Strength In Numbers: Finding And Developing Bioinformaticians.”
  *Front Line Genomics*. 13 October 2014\.

* “Kelso: For barbecue, this Texan-turned-Canadian goes to great
  lengths.” *Austin American-Statesman*. 27 July 2014\.

* “As Biomedical Researchers Face Tough Job Market, Experts Offer Advice
  and Propose Changes.” GenomeWeb Daily News. 5 June 2014\.

* “101 questions with a bioinformatician \#4: Michael Hoffman.” ACGT. 20
  April 2014\.

* “Meet Michael Hoffman.” Princess Margaret Cancer Foundation Blog. 16
  January 2014\.

## Citizenship and immigration

* United States of America: citizen.

* Canada: permanent resident.
